src/ios/FCMPlugin.m

```diff
@@ -17,6 +17,11 @@ @implementation FCMPlugin
 
 static NSString *notificationCallback = @"FCMPlugin.onNotificationReceived";
 static NSString *tokenRefreshCallback = @"FCMPlugin.onTokenRefreshReceived";
+
+static NSString *FCM_SERVER_CONNECTION = @"@gcm.googleapis.com";
+static NSString *FCM_PROJECT_SENDER_ID;
+static long long ttl = 900;
```

Comment:
why `long long` ?

Reply:
I think I ran into an error and had to change it to that. Let me fix it.